pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -639,3 +752,63 @@ func updateCnsRegisterVolume(ctx context.Context, client client.Client,
 	}
 	return err
 }
+
+func getPatchData(oldObj, newObj interface{}) ([]byte, error) {
+	oldData, err := json.Marshal(oldObj)
+	if err != nil {
+		return nil, fmt.Errorf("marshal old object failed: %v", err)
+	}
+	newData, err := json.Marshal(newObj)
+	if err != nil {
+		return nil, fmt.Errorf("marshal new object failed: %v", err)
+	}
+	patchBytes, err := jsonpatch.CreateMergePatch(oldData, newData)
+	if err != nil {
+		return nil, fmt.Errorf("CreateMergePatch failed: %v", err)
+	}
+	return patchBytes, nil
+}
+func patchStoragePolicyUsage(ctx context.Context, cnsOperatorClient client.Client,
+	oldObj *storagepolicyusagev1alpha1.StoragePolicyUsage,
+	newObj *storagepolicyusagev1alpha1.StoragePolicyUsage) error {
+	log := logger.GetLogger(ctx)
+	patch, err := getPatchData(oldObj, newObj)
+	if err != nil {
+		log.Errorf("error fetching PatchData StoragePolicyUsage CR. err: %v", err)
+		return err
+	}
+	patch, err = addResourceVersion(patch, oldObj.ResourceVersion)
+	if err != nil {
+		log.Errorf("applying ResourceVersion to patch data failed: %v", err)
+		return err
+	}
+	rawPatch := client.RawPatch(apitypes.MergePatchType, patch)
+	err = cnsOperatorClient.Patch(ctx, oldObj, rawPatch)
+	log.Infof("Patching the StoragePolicyUsageCR %q on namespace: %q with the data: %v",
```

Comment:
could you change "%v" to "%+v"?

Reply:
Done. And also updated the variable to rawPatch(which is the object I wanted to print)